src/main/java/core/basesyntax/io/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.io;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> info = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                info.add(line);
+            }
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
```

Comment:
The exception message 'Can`t read file' is not very informative. It's better to include the file name that couldn't be read to provide more context, which would be helpful for debugging.